wire-ios-data-model/Support/Sourcery/generated/AutoMockable.manual.swift

```diff
@@ -0,0 +1,326 @@
+// Generated using Sourcery 2.2.4 — https://github.com/krzysztofzablocki/Sourcery
+// DO NOT EDIT
+
+//
```

Comment:
We could get rid of this header here, right?
```suggestion
//
```